geostore/check.py

```diff
@@ -6,9 +6,11 @@ class Check(Enum):
     CHECKSUM = "checksum"
     DUPLICATE_OBJECT_KEY = "duplicate asset name"
     FILE_NOT_FOUND = "file not found in staging or storage"
+    INVALID_STAC_ROOT_TYPE = "root type should be catalog or collection"
```

Comment:
"must" rather than "should" might be helpful here.

Reply:
Updated, thanks.